src/ape/managers/project/manager.py

```diff
@@ -447,16 +447,18 @@ def _get_contract(self, name: str) -> Optional[ContractContainer]:
 
         return None
 
-    # @property
-    # def meta(self) -> PackageMeta:
-    #     return PackageMeta(**self.config_manager.get_config("ethpm").serialize())
+    @property
+    def meta(self) -> PackageMeta:
+        return PackageMeta(**self.config_manager.get_config("ethpm").serialize())
+
+    def publish_manifest(self):
+        manifest = self.manifest.dict()
+        if not manifest["name"]:
+            raise ProjectError("Need name to release manifest")
+        if not manifest["version"]:
+            raise ProjectError("Need version to release manifest")
+        breakpoint()
 
-    # def publish_manifest(self):
-    #     manifest = self.manifest.dict()
-    #     if not manifest["name"]:
-    #         raise ProjectError("Need name to release manifest")
-    #     if not manifest["version"]:
-    #         raise ProjectError("Need version to release manifest")
     #     TODO: Clean up manifest and minify it
     #     TODO: Publish sources to IPFS and replace with CIDs
```

Comment:
Note that this would require looping through all the sources and publishing them to IPFS as well, then updating all of the links in the package from local links to global ones.

Also side-note is that eventually `DependencyAPI` should store everything as IPFS content locally to make it super easy to publish